platformLibs/build.gradle

```diff
@@ -22,6 +22,12 @@ buildscript {
     }
 
     dependencies {
+        constraints {
+            classpath('org.jetbrains.kotlin:kotlin-reflect') { version { strictly kotlinVersion } }
+            classpath('org.jetbrains.kotlin:kotlin-stdlib-common') { version { strictly kotlinVersion } }
+            classpath('org.jetbrains.kotlin:kotlin-stdlib') { version { strictly kotlinVersion } }
+        }
+
```

Comment:
Is it possible to move this `constraints { }` block to some method to `build-tools` or to the `gradle/kotlinGradlePlugin.gradle`?

Reply:
I'm not sure it can be easily done without introducing another file similar to `gradle/kotlinGradlePlugin.gradle`. `gradle/kotlinGradlePlugin.gradle` implies to many side effects and there are several projects involved that do not depend on `build-tools`. 

I think such refactoring can be postponed till the general approach proves to be useful.